pkg/model/deployment_chain.proto

```diff
@@ -52,9 +52,30 @@ message ChainNode {
     ChainDeploymentRef deployment_ref = 2;
 }
 
+// ChainBlockStatus represents the current status of a block in deployment chain.
+enum ChainBlockStatus {
+    // DEPLOYMENT_BLOCK_PENDING means the block's nodes just initialized,
+    // all block nodes do not have its deploymentRef created.
```

Comment:
to be correct: all node deployments have not been initialized oe still at PENDING